### PR TITLE
search with case-insensitive email value

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
#### Which User story does this relate to (link)?
[COG-1342])

#### Brief feature description
- Search with case insensitive email

### CODE REVIEW

**Comment by Sushmareddy-OSI** [comment]
retest this please

File: lib/elastic/user_query_builder.rb [hunk 1]
```diff
@@ -55,7 +55,8 @@ def self.query_leaves(query)
         { conjunction: 'AND', query: { terms: { 'office_id.keyword': value } } } unless value.empty?
       end,
       email: lambda do |value|
-        { conjunction: 'AND', query: { match: { 'email.keyword': value } } } unless value.empty?
+        { conjunction: 'AND', query: { match: { 'email.keyword': value.downcase } } } unless value
```
Comment (kelfink) [inline lib/elastic/user_query_builder.rb]: interesting need for downcast here, yet upcase in the other field?  Could they both be upcase ?  Just curious.

Comment (kelfink) [reply lib/elastic/user_query_builder.rb]: If they *could* both be the same casing, maybe we should be doing that for all searched fields...

Comment (Sushmareddy-OSI) [reply lib/elastic/user_query_builder.rb]: firstName and LastName are already case-insenstive, thats how the dora is accepting..

**Pull request merged**